src/acceptance-test/java/org/zalando/nakadi/repository/db/TimelineDbRepositoryTest.java

```diff
@@ -0,0 +1,143 @@
+package org.zalando.nakadi.repository.db;
+
+import java.util.Date;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+import java.util.stream.Collectors;
+import java.util.stream.LongStream;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.dao.DuplicateKeyException;
+import org.zalando.nakadi.domain.EventType;
+import org.zalando.nakadi.domain.Storage;
+import org.zalando.nakadi.domain.Timeline;
+import org.zalando.nakadi.exceptions.DuplicatedEventTypeNameException;
+import org.zalando.nakadi.exceptions.InternalNakadiException;
+import org.zalando.nakadi.utils.TestUtils;
+
+public class TimelineDbRepositoryTest extends AbstractDbRepositoryTest {
```

Comment:
I totally fine with this test, because it helped me to fix errors with sql in this home-made ORM (I renamed fields, and tests showed me 2 places where sql was not correct). 
So they already saved time for me.